### ISSUE TITLE
UART sensors are no longer autodetected

### ISSUE THREAD

**Issue by ddemidov** [ev3dev/ev3dev#461]
It seems that after upgrading to the latest kernel (
```
$ uname -a
Linux ev3dev 3.16.7-ckt21-9-ev3dev-ev3 #1 PREEMPT Tue Dec 15 15:16:17 CST 2015 armv5tejl GNU/Linux
```
) UART sensors are no longer autodetected. For example, connecting EV3 infrared sensor emits the following message in `dmesg`:
```
[  512.127214] lego-port port3: Added new device 'in4:ev3-uart-host'
```
but the sensor is not listed in `/sys/class/lego-sensor/`.

Same thing happens with EV3 color sensor:
```
[  739.804522] lego-port port0: Added new device 'in1:ev3-uart-host'
```
Again, no node in `/sys/class/lego-sensor/`.

Analog sensors (tested EV3 touch sensor) and i2c sensors (tested ms-absolute-imu and pixycam adapter) work fine.

**Comment by ddemidov** [comment]
Also tried rebooting with and without sensors connected -- same result.

**Comment by dlech** [comment]
Should be fixed in `ev3-system 5.1.0`. I'm pushing it to the server, so please wait a minute or two, then `apt-get update; apt-get upgrade`.

**Comment by ddemidov** [comment]
It works now, thanks for the quick fix!
